### DISCUSSION TITLE
Ubuntu DIiaNN1.9.1 Erorr after create Report-first-pass.manifest.txt

### DISCUSSION THREAD

**Discussion by huangcx1539** [vdemichev/DiaNN#1122] [Q&A] [answered]
The DiaNN log file didnot show any erorr message, the job was stopped after save Report-first-pass.manifest.txt
and it show:
terminate called after throwing an instance of 'std::length_error'
  what():  basic_string::_M_create

![1723100967627](https://github.com/user-attachments/assets/932ba209-467e-4265-86b9-28eb544d937c)

**Comment by huangcx1539** [comment]
![1723101987450](https://github.com/user-attachments/assets/2657270f-a8ba-4b24-9eff-7a01c1cee807)

**Reply by vdemichev** [reply, marked as answer]
Hi, this is a known error, will be fixed in 1.9.2.
For now, please don't enable matrix output on Linux, i.e. don't use --matrices.

Best,
Vadim
